content/docs/languages-sdks/yaml/yaml-language-reference.md

```diff
@@ -116,23 +116,34 @@ There are at least two reasons to explicitly define providers in YAML, or explic
 1. Using explicit versions enables pinning the dependencies used, a technique used to improve build reliability.
 2. Using explicit providers enables controlling the options for providers used by each resource, as described in [Unlock Programmatic Control by Disabling Default Providers](/blog/disable-default-providers/).
 
-#### Resource version
+If a specific provider version is necessary, we prefer the approaches in the order described here.
```

Comment:
Want users to fall into "pit of success" here, a program with default providers is more likely to be correct than one with explicit providers passed to multiple resources or setting these options on individual resources.